docs/source/api/pyjobshop.rst

```diff
@@ -70,8 +70,34 @@ PyJobShop
 
    .. autoclass:: Result
 
-   .. autoclass:: SolveStatus
-      :members:
+   .. class:: SolveStatus
```

Comment:
Sphinx's autoapi doesn't work well with inheriting from `str`. So hardcoding here is the best option. I don't expect SolveStatus to change in the near future.